### DISCUSSION TITLE
惊喜！！！遇到思路一样的作者了。

### DISCUSSION THREAD

**Discussion by AimLuo** [ofajs/discussion#5] [General]
我是通过 parcel + ejs 的方式来实现的，ejs仅作为入口文件，跨框架的调用通过自定义事件来触发。整体比较简陋，不过思路跟作者差不多
另外 建议把 tailwind css 的使用方式集成一下。

**Comment by kirakiray** [comment]
是的，你说的这种有点像简化版的微前端，我知道的还有用 web container 去做的。

好的，已经 tailwind css 的支持加入 todo。
